src/EditorFeatures/Core.Wpf/SignatureHelp/Controller_NavigationKeys.cs

```diff
@@ -55,12 +55,11 @@ private bool ChangeSelection(Action computationAction)
                 computationAction();
                 return true;
             }
-            else
-            {
-                // Dismiss ourselves and actually allow the editor to navigate.
-                DismissSessionIfActive();
-                return false;
-            }
```

Comment:
i don't understand the fix. the original code seems right (and docs why we have this behavior).